lexers/s/slurm.go

```diff
@@ -0,0 +1,18 @@
+package s
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// Slurm lexer. Lexer for (ba|k|z|)sh Slurm scripts.
+var Slurm = internal.Register(MustNewLexer(
```

Comment:
This lexer inherits `analyse_text` from [BashLexer](https://github.com/pygments/pygments/blob/a590ac5ea7c00a41e253834306bfa19e38349c0b/pygments%2Flexers%2Fshell.py#L123).

Reply:
Good catch :+1: